packages/framework/build.xml

```diff
@@ -126,7 +126,7 @@
         </exec>
     </target>
 
-    <target name="create-domains-data" depends="create-domains-db-functions" description="Creates domains data for newly configured domains.">
+    <target name="create-domains-data" depends="create-domains-db-functions" description="Creates domains data for newly configured domains. You should run it when you modify domains.yml">
```

Comment:
why the use of 2.person ?

Reply:
How would you formulate it?

Reply:
There are more second-person usages, usually in the form of an imperative (eg. *Always run 'standards' to be sure there are no further violations.*) or in terms *your DB* / *your project*